salt/modules/freebsd_sysctl.py

```diff
@@ -30,7 +30,7 @@ def __virtual__():
 
 
 def _formatfor(name, value, config, tail=""):
-    if config == "/boot/loader.conf":
+    if config == "/boot/loader.conf.local":
```

Comment:
**question** From https://www.freebsd.org/cgi/man.cgi?loader.conf(5) and https://forums.freebsd.org/threads/boot-loader-conf-vs-boot-loader-conf-local.60259/ it appears that `/boot/loader.conf` should not be changed during an upgrade. Is that an incorrect understanding?